kratos/python_scripts/apply_periodic_boundaray_condition_process.py

```diff
@@ -0,0 +1,115 @@
+import KratosMultiphysics as KratosMultiphysics
+
+def Factory(settings, Model):
+    if(type(settings) != KratosMultiphysics.Parameters):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+    return ApplyPeriodicBoundaryConditionProcess(Model, settings["Parameters"])
+
+## All the processes python should be derived from "Process"
+class ApplyPeriodicBoundaryConditionProcess(KratosMultiphysics.Process):
+    """This class is used in order to impose a rigid body movement in a certain region of the problem
+
+    This class constructs the model parts containing the constrains that enforce the rigid body movement
+    Only the member variables listed below should be accessed directly.
+
+    Public member variables:
+    Model -- the container of the different model parts.
+    settings -- Kratos parameters containing solver settings.
+    """
+
+    def __init__(self, Model, settings ):
+        """ The default constructor of the class
+
+        Keyword arguments:
+        self -- It signifies an instance of a class.
+        Model -- the container of the different model parts.
+        settings -- Kratos parameters containing solver settings.
+        """
+        KratosMultiphysics.Process.__init__(self)
+
+        default_settings = KratosMultiphysics.Parameters("""
+        {
+            "help"                        : "This process uses LinearMasterSlaveConstraint in order to impose a periodic boundary condition on the given submodelparts. The process takes the first provided submodelpart as master and the second as slave.",
```

Comment:
"help" should not be a parameter. There is already a python feature to get help from the docstring.

Reply:
I think those were added by @loumalouomega and @roigcarlo for the processes-documentation right?

Reply:
Yep